examples/nextjs/src/components/my-page.js

```diff
@@ -27,7 +26,7 @@ export function MyPage({ data, nav }) {
   }
   
   return (
-    <DotExperimentsProvider config={experimentConfig} >
+    <DotExperimentsProvider config={experimentConfig}>
```

Comment:
This is actually not working... if in your nextjs app don't have the env var `NEXT_PUBLIC_EXPERIMENTS_API_KEY` and you running it crashed.

So 2 things:

1. It should not crash
2. If we don't have the env vars of the experiments we should not add any of the experiments components or providers.

Please fix those two because we already have customers using this examples without experiments.

Reply:
Let me fix this.

Reply:
Handled.